include/GPUInfoProvider.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <lug/Config.hpp>
+#include <lug/Graphics/Renderer.hpp>
+#include <lug/Graphics/Vulkan/API/RTTI/Enum.hpp>
+#include <lug/Graphics/Vulkan/Renderer.hpp>
+
+#if defined(LUG_SYSTEM_WINDOWS)
+    #pragma warning(push)
+    #pragma warning(disable : 4003)
+#endif
+#include <json/json.hpp>
+#if defined(LUG_SYSTEM_WINDOWS)
+    #pragma warning(pop)
+#endif
+
+namespace GPUInfoProvider {
+
+    nlohmann::json getJSONVulkanInfo(lug::Graphics::Vulkan::PhysicalDeviceInfo &physicalDeviceInfo);
+};
```

Comment:
No `;` after a namespace declaration.
Put a comment with the name of the namespace.